Smarty/templates/Header.tpl

```diff
@@ -11,17 +11,21 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
 <html>
 <head>
-	<meta http-equiv="Content-Type" content="text/html; charset={$LBL_CHARSET}">
+	<meta http-equiv="Content-Type" content="text/html; charset={$APP.LBL_CHARSET}">
```

Comment:
This change is not related to your extension and it is incorrect as coreBOS does not have that string in its translation files anymore. Please eliminate this change.